java-components/cli/src/main/java/io/github/redhatappstudio/jvmbuild/cli/builds/BuildLogsCommand.java

```diff
@@ -117,6 +119,9 @@ public void run() {
                 }
                 buildNumbers.add(i);
             }
+            for (int i = 0; i < theBuild.getStatus().getBuildAttempts().size(); ++i) {
+                buildNumbers.add(i);
+            }
```

Comment:
Can you explain this change (compared to the pre-existing loop above). I didn't think I changed this code?

Reply:
If all the pipeline runs have been deleted then you end up with an empty list.

I probably should have deleted the code above, build attempts should be the definitive record of how many builds have been done.